exercise-solutions/quiz-game/part-04/src/index.jsx

```diff
@@ -2,16 +2,14 @@ import React from "react";
 import ReactDOM from "react-dom";
 import {Match} from "./match";
 
-
 const App = () => {
         return (
-            <div>
+            <>
                 <Match/>
-            </div>
+            </>
         );
```

Comment:
couldn't return `<Match/>` directly here?

Reply:
probably. I dont know. You added the div. I just changde it to a fragment because it fucked up my flex box. I can test it.

Reply:
worked fine without it. I deleted it. =)